pkgs/servers/computing/slurm/default.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, fetchurl, pkgconfig, curl, python, munge, perl, pam, openssl,
-  ncurses, mysql, gtk }:
+  ncurses, mysql, gtk, lua, hwloc, numactl, extraDeps ? [] }:
```

Comment:
Mainly to support arbitrary plugins like auks or super-computer ones with a simple override.

extraDeps is passed as a buildInput